content/en/dashboards/ddsql_editor/reference/_index.md

```diff
@@ -0,0 +1,61 @@
+---
+title: DDSQL Reference
+---
+
+{{< callout url="https://google.com">}}
+DDSQL is in private beta.
+{{< /callout >}}
+
+## Overview
+
+DDSQL is a query language for Datadog data. It implements several standard SQL functions, such as `SELECT`, along with new functions for working with Datadog tags, metrics, and so on. For example, [the `AGGR` statement][5], which aggregates metrics data, is unique to DDSQL.
+
+You can use DDSQL to query your infrastructure in the [DDSQL Editor][2] in Datadog.
+
+DDSQL uses a [schema on read](#schema-on-read) approach to allow queries against unstructured data, such as [tags][1]. Each query is executed within a [session](#sessions) that can be configured with environment variables.
+
+## Supported SQL syntax
+
+SQL is broken into five different categories of statements. The table below indicates which categories are supported by DDSQL.
+
+| Category   | Examples    | Support      |
+|------------|-------------|--------------|
+| DQL (Data Query Language)   | `SELECT`, `AGGR` (DDSQL alternative to `SELECT`) | Supported    |
+| DML (Data Modification Language)  | `INSERT`, `UPDATE`, `DELETE`   | Limited: Data is not persisted across sessions |
+| DDL (Data Description Language)  | `CREATE`   | Limited: Data is not persisted across sessions |
+| DCL (Data Control Language)        | `GRANT`, `REVOKE`   | Not supported       |
+| TCL (Transaction Control Language) | `BEGIN`, `END`, `ROLLBACK`  | Not supported     |
+
+## Usage details
+
+{{< whatsnext desc="For usage details, see the relevant documentation:" >}}
+   {{< nextlink href="dashboards/ddsql_editor/reference/statements" >}}Statements (such as SELECT){{< /nextlink >}}
+   {{< nextlink href="dashboards/ddsql_editor/reference/data_types" >}}Data types{{< /nextlink >}}
+   {{< nextlink href="dashboards/ddsql_editor/reference/expressions_and_operators" >}}Expressions and operators{{< /nextlink >}}
+   {{< nextlink href="dashboards/ddsql_editor/reference/scalar_functions" >}}Scalar functions (one computed value per row){{< /nextlink >}}
+   {{< nextlink href="dashboards/ddsql_editor/reference/aggregation_functions" >}}Aggregation functions (one computed value per query or group){{< /nextlink >}}
+   {{< nextlink href="dashboards/ddsql_editor/reference/window_functions" >}}Window functions{{< /nextlink >}}
+   {{< nextlink href="dashboards/ddsql_editor/reference/tags" >}}Querying tags{{< /nextlink >}}
+{{< /whatsnext >}}
+
+## Sessions
```

Comment:
same here- this is an advanced detail that should be covered later.

Reply:
@cangencer The struggle here is that I don't want to bury this under SET or SHOW, since it's broader information than just how to use those statements. It could go at the top of the statements page, if you think that's better? 

Personally, as long as the content of a page goes from "less advanced" to "more advanced," I don't consider this content out of scope on a reference index page covering what DDSQL is and how it works in general. It's not high level information, but it's broadly applicable information that affects every query. But happy to move it to the statements page or another location that works better!

Reply:
Fair enough - my take was that this is a rather advanced feature and most users will want to just "query stuff". It could go under "concepts" or something like that (same with schema-on-read)

Reply:
I would just remove it, i don't think this is necessary, and is more confusing than anything